### PR TITLE
[DE] Add Support for further German Sentences for the Light Domain

### PR DESCRIPTION
**Pull request by Johnson145**

### CODE REVIEW

**Comment by Johnson145** [comment]
@easterapps @mib1185 May I kindly ask for a review on this PR as well?

**Review by kroimon — APPROVED** [review]
Sounds good to me, I like to give short, robotic commands to voice assistants sometimes 😅 German efficiency, I guess...

**Comment by Johnson145** [comment]
@kroimon Thanks! May I ask you to merge the PR? I think I can't do it on my own even though you have approved it already.

File: tests/de/light_HassLightSet.yaml
Comment (mib1185) [inline tests/de/light_HassLightSet.yaml:1]: We need some more tests to get all new possible permutations covered

**Review by mib1185 — CHANGES_REQUESTED** [review]
HI @Johnson145 
thanks for these additions 👍  just one comment about tests

File: tests/de/light_HassLightSet.yaml
Comment (Johnson145) [reply tests/de/light_HassLightSet.yaml:1]: I added new test sentences and slightly modified the general syntax again to match all of the new tests. Would be great, if you can review again. Let me know if you're still missing something.

**Review by mib1185 — CHANGES_REQUESTED** [review]
HI @Johnson145 
overall I think this looks pretty well, but would be great we could have this logic also for setting a color. Thx 👍

**Comment by Johnson145** [comment]
@mib1185 I've just extended the PR to cover similar sentences for changing the color of a light. Looking forward to a final review.

**Comment by Johnson145** [comment]
@mib1185 Updated the branch one more time based on the `main`. I guess you're doing this in your freetime, too, and I don't want to bother you too much, but can you please have another look whether we can merge this by now?

**Review by mib1185 — APPROVED** [review]
Hi @Johnson145 
that's correct, this is all within my spare time 😉 
This PR looks good, thanks for this 👍

**Pull request merged**